ui/app/components/license-banners.js

```diff
@@ -9,11 +17,25 @@
  * @param {string} expiry - RFC3339 date timestamp
  */
 
-import Component from '@glimmer/component';
-import isAfter from 'date-fns/isAfter';
-import differenceInDays from 'date-fns/differenceInDays';
-
 export default class LicenseBanners extends Component {
+  @service version;
+
+  @tracked currentVersion = this.version.version;
```

Comment:
This should be able to be a getter, we aren't expecting to change this from within the component
```suggestion
  get currentVersion() {
  return this.version.version;
  }
```